### DISCUSSION TITLE
Running lambda from queue locally

### DISCUSSION THREAD

**Discussion by jacobweberbowery** [getlift/lift#264] [General]
When using [queues](https://github.com/getlift/lift#queue), where the lambda is defined inside the queue's `worker` field, is it possible to run the lambda locally, like you would with `sls invoke local`?

**Comment by mnapoli** [comment]
Yes you can use `sls invoke local` to invoke the queue's worker function. If you run `sls info` you should be able to see the name of the worker function. This worker function is a standard Serverless Framework function, so all classic commands work with it.

**Reply by jacobweberbowery** [reply]
I could've sworn I tried that, but you're right — `queueWorker` is the name. Thank you!
